cypress/e2e/StatusList.spec.cy.ts

```diff
@@ -0,0 +1,41 @@
+const EXPAND_INTERFACES_LIST_TEST_ID = '#expand-interfaces-list0';
```

Comment:
It would be cleaner to keep the constants in a separate file.

Reply:
fixed